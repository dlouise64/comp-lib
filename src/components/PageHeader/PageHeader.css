.pageHeader {
  color: rgb(51, 51, 51);
  padding-bottom: 20px;
  border-bottom: 2px solid rgb(51, 51, 51);
}
