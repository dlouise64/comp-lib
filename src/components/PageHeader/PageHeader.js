import React from 'react';
import PropTypes from 'prop-types';

import './PageHeader.css';

class PageHeader extends React.Component {
  render() {
    const {message} = this.props;
    return (
      <h1 className="pageHeader">{message}</h1>
    );
  }
}

PageHeader.propTypes = {
  /** the message to display in the title */
  message: PropTypes.string.isRequired
};

PageHeader.defaultProps = {
}

export default PageHeader;
