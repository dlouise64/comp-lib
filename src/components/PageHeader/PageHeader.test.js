import React from 'react';
import renderer from 'react-test-renderer';
import PageHeader from './PageHeader';

test('<PageHeader /> renders correctly', () => {
  const tree = renderer.create(<PageHeader
    message="The Title"
  />).toJSON();
  expect(tree).toMatchSnapshot();
});
