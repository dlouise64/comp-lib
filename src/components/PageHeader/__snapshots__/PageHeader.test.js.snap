// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PageHeader /> renders correctly 1`] = `
<h1
  className="pageHeader"
>
  The Title
</h1>
`;
